### PR TITLE
Add v2 schema

### PR DESCRIPTION
**Pull request by Pete-Y-CS**
Might wants some UI considerations, the type radio is behaving weirdly for me, and specific options and wording might want tweaking but here's a first start

### CODE REVIEW

File: src/lib/sketch/FormV2.svelte [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  import { prettyPrintMeters } from "lib/maplibre";
+  import type { InterventionProps, Schemes } from "types";
+  import { schemeName } from "./config";
+  import type { Writable } from "svelte/store";
+  import type { FeatureProps } from "scheme-sketcher-lib/draw/types";
+
+  export let gjSchemes: Writable<Schemes>;
+  export let props: FeatureProps<InterventionProps>;
+
+  props.v2 = props.v2 || {
```
Comment (dabreegster) [inline src/lib/sketch/FormV2.svelte:21]: You can do `||=`

File: src/lib/sketch/FormV2.svelte [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+<Select
+  label="Scheme"
+  choices={Object.values($gjSchemes.schemes).map((scheme) => [
+    scheme.scheme_reference,
+    schemeName(scheme),
+  ])}
+  bind:value={props.scheme_reference}
+/>
+
+<Radio
```
Comment (dabreegster) [inline src/lib/sketch/FormV2.svelte:60]: To fix TS errors here, wrap all of this in a `{#if props.v2}`. It's kind of annoying, but the type checker doesn't understand the `props.v2 ||= {}` guarantee above. This approach has been working fine in PipelineForm

File: src/lib/sketch/FormV2.svelte [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  ]}
+  inlineSmall
+  required
+  bind:value={props.v2.intervention_type}
+/>
+
+<TextArea label="Description" bind:value={props.description} />
+
+<Checkbox checked={props.v2.for_cyclists}>Is this for cyclists?</Checkbox>
+<Checkbox checked={props.v2.for_pedestrians}>Is this for pedestrians?</Checkbox>
```
Comment (dabreegster) [inline src/lib/sketch/FormV2.svelte:78]: Let's check wording -- walking/wheeling?

File: src/stores.ts [hunk 4]
```diff
@@ -29,5 +29,5 @@ function defaultSchema(): Schema {
     return "pipeline";
   }
 
-  return "v1";
+  return "v2";
```
Comment (dabreegster) [inline src/stores.ts:32]: Ooh exciting, we're changing the default!

File: src/types.ts [hunk 5]
```diff
@@ -43,6 +44,9 @@ export interface PipelineScheme extends PipelineBudget, PipelineTiming {
   scheme_description: string;
 }
 
+export interface V2Scheme {
```
Comment (dabreegster) [inline src/types.ts:47]: // Empty as a marker for detecting v2 and for future possible additions

File: src/types.ts [hunk 6]
```diff
@@ -122,3 +127,11 @@ export interface PipelineIntervention extends PipelineBudget, PipelineTiming {
   // TODO new / existing / upgrade existing?
   // TODO for routes, ltn120 type: fully protected, light segregation, off-carriageway, shared-use, dedicated footpath. minimum width?
 }
+
+export interface V2Intervention {
+  // These 3 fields are v1 schema
```
Comment (dabreegster) [inline src/types.ts:132]: Dunno what this comment means

**Comment by dabreegster** [comment]
- [ ] Check the categories/schema
- [ ] Check the wording
- [ ] Update browse -- make sure everything works with local sketches, change intervention popups, coloring, etc
- [ ] New integration test

**Comment by dabreegster** [comment]
@Pete-Y-CS, minus the tests, I think this is close to ready. I adjusted wording of things and added browse support. Please try it out e2e and let me know if anything should change.

**Comment by dabreegster** [comment]
Switched to a radio for intended uses and added a test. Ready for review!

**Review by Pete-Y-CS — COMMENTED** [review]
i can't approve but lgtm

**Pull request merged**